src/clarity-angular/nav/_nav.clarity.scss

```diff
@@ -49,8 +49,7 @@ $clr-active-nav-link-shadow: 0 -3px 0 $action-blues-dark-midtone inset;
             }
 
             &:hover,
-            &.active,
-            &:focus {
```

Comment:
@jeeyun: are we removing the focus states altogether? Will this mean that the tabs revert back to the browser default focus states and wont the browser default states still show when the user hits the back button? 

What if we programmatically remove the focus when the user switches a tab? (btw eudes didn't want us to use blur to remove focus states because it resets the focus to the top. so we should check with him once on this if we decide to go ahead with this approach) cc: @youdz